jasmine-es6-promise-matchers.js

```diff
@@ -40,7 +40,16 @@
     RESOLVED: 'resolved',
   };
 
+  function isError(value) {
+    return typeof value === 'object' && value instanceof Error;
```

Comment:
Why is the `typeof value === 'object'` necessary? (Why isn't `value instanceof Error` sufficient?)

Reply:
You're right! Is not needed indeed, the browser handles invalid values as well.